### PR TITLE
week2 task

### PR DESCRIPTION
**Pull request by saurabhmishraepam**

### CODE REVIEW

File: src/main/resources/liquibase-changeLog.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+   http://www.liquibase.org/xml/ns/dbchangelog
+   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet author="John" id="someUniqueId">
+        <addColumn tableName="users">
```
Comment (ashoksuryap) [inline src/main/resources/liquibase-changeLog.xml]: Do we need this file?

File: src/main/java/com/epam/productsreview/config/ApplicationConfig.java [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
 @Configuration
 public class ApplicationConfig {
-
-
+    @Autowired
+    private DataSource dataSource;
+    @Bean
+    public SpringLiquibase liquibase() {
+        SpringLiquibase liquibase = new SpringLiquibase();
+        liquibase.setChangeLog("classpath:liquibase-changeLog.xml");
```
Comment (ashoksuryap) [inline src/main/java/com/epam/productsreview/config/ApplicationConfig.java:20]: Need to point liquibase-outputChangeLog.xml file?

File: src/main/resources/liquibase-outputChangeLog.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="saurabh (generated)" id="1551725935105-1">
+        <createTable tableName="hibernate_sequence">
```
Comment (ashoksuryap) [inline src/main/resources/liquibase-outputChangeLog.xml]: Do we need this table?

File: src/main/java/com/epam/productsreview/config/ApplicationConfig.java [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
  * Created by saurabh on 26/2/19.
  */
 @Configuration
 public class ApplicationConfig {
-
-
+    @Autowired
+    private DataSource dataSource;
+    @Bean
+    public SpringLiquibase liquibase() {
```
Comment (ashoksuryap) [inline src/main/java/com/epam/productsreview/config/ApplicationConfig.java:18]: Shouldn't this bean recreates your database every time you restart your app?

File: src/main/resources/liquibase-changeLog.xml (hunk 1 above)
Comment (saurabhmishraepam) [reply src/main/resources/liquibase-changeLog.xml]: I have removed this.

File: src/main/java/com/epam/productsreview/config/ApplicationConfig.java (hunk 2 above)
Comment (saurabhmishraepam) [reply src/main/java/com/epam/productsreview/config/ApplicationConfig.java:20]: removed it as this was only to test the behaviour

File: src/main/resources/liquibase-outputChangeLog.xml (hunk 3 above)
Comment (saurabhmishraepam) [reply src/main/resources/liquibase-outputChangeLog.xml]: removed it

File: src/main/java/com/epam/productsreview/config/ApplicationConfig.java (hunk 4 above)
Comment (saurabhmishraepam) [reply src/main/java/com/epam/productsreview/config/ApplicationConfig.java:18]: removed it

**Review by ashoksuryap — APPROVED** [review]

**Pull request merged**